### ISSUE TITLE
Accessibilty ARIA issue

### ISSUE THREAD

**Issue by brusand** [jpuri/react-draft-wysiwyg#1372]
Hi, i have an issue with the option toolbar when i check with axe.

Due to the line  aria-selected={active} in src/components/options.js
i have this issue    
    Expected the HTML found at $('#rdw-wrapper-5219 > .rdw-editor-toolbar.rsw-toolbar[aria-label="rdw-toolbar"] > .rdw-inline-wrapper[aria-label="rdw-inline-control"] > .rdw-option-wrapper[title="Bold"]') to have no violations:

    <div class="rdw-option-wrapper" aria-selected="false" title="Bold"><img alt="" src="/assets/img/bold.svg"></div>

    Received:

    "Elements must only use allowed ARIA attributes (aria-allowed-attr)"

    Fix any of the following:
      ARIA attribute is not allowed: aria-selected="false"

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.5/aria-allowed-attr?application=axeAPI

How to fixed it or customized it ?
Thx
Bruno

**Comment by sridharvemulapalli** [comment]
Facing the same, Is there any update on the same?
